src/main/navigation/navigation_fw_launch.c

```diff
@@ -137,6 +137,30 @@ static void applyFixedWingLaunchIdleLogic(void)
     }
 }
 
+static inline bool isFixedWingLaunchMaxAltitudeReached(void)
+{
+    return (navConfig()->fw.launch_max_altitude > 0) && (getEstimatedActualPosition(Z) >= navConfig()->fw.launch_max_altitude);
+}
+static inline bool isLaunchModeMinTimeElapsed(float timeSinceLaunchMs)
+{
+    return timeSinceLaunchMs > navConfig()->fw.launch_min_time;
+}
+
+static inline bool isLaunchModeMaxTimeElapsed(float timeSinceLaunchMs)
+{
+    return timeSinceLaunchMs >= navConfig()->fw.launch_timeout;
+}
+
+static inline bool isLaunchModeFinishedByPilot()
+{
+    return (ABS(rcCommand[ROLL]) > rcControlsConfig()->pos_hold_deadband) || (ABS(rcCommand[PITCH]) > rcControlsConfig()->pos_hold_deadband);
+}
+
+static inline bool checkFixedWingLaunchState(float timeSinceLaunchMs)
```

Comment:
The name is confusing. Possibly `isFixedWingLaunchCompleted` is a better name?

Reply:
Followed your suggestion an updated Cli documentation too.